app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -47,7 +47,14 @@ def raw_delete_volume
   end
 
   def validate_attach_volume
-    validate_volume_available
+    msg = validate_volume_available
+    return {:available => msg[:available], :message => msg[:message]} unless msg[:available]
+
+    if status == "in-use"
+      return validation_failed("Attach Volume", "Can't attach non-shareable volume that is in use.") unless multi_attachment
```

Comment:
Looks a little strange to split the conditionals, can this be just `if status == "in-use" && !multi_attachment` ?

Reply:
Yeah I only use unless for a single negative conditional, anything more complex gets `if`